src/packages.lisp

```diff
@@ -9,7 +25,8 @@
 (defpackage #:cl-raster/core
   (:use #:cl)
   (:export #:render)
-  (:local-nicknames (#:scene #:cl-raster/scene)))
+  (:local-nicknames (#:scene #:cl-raster/scene)
+                    (#:vectors #:3d-vectors)))
```

Comment:
Maybe a shorter nickname would be better? `vec` or even `v`, since vectors are used a lot.